kofta/src/app/components/UserProfile.tsx

```diff
@@ -81,40 +82,51 @@ export const UserProfile: React.FC<UserProfileProps> = ({
         >
           <Avatar src={profile.avatarUrl} />
         </div>
-        {me?.id === profile.id ? (
-          <div>
-            <Button
-              onClick={() => {
-                setEditProfileModalOpen(true);
-              }}
-              variant="small"
-            >
-              {t("pages.viewUser.editProfile")}
-            </Button>
-          </div>
-        ) : null}
-        {me?.id === profile.id ||
-        userProfile.youAreFollowing === null ||
-        userProfile.youAreFollowing === undefined ? null : (
-          <div>
-            <Button
+        <div className="flex items-center">
+          <Button
               onClick={() => {
-                wsend({
-                  op: "follow",
-                  d: {
-                    userId: profile.id,
-                    value: !youAreFollowing,
-                  },
-                });
-                setYouAreFollowing(!youAreFollowing);
-                onFollowUpdater(setCurrentRoom, me, profile);
+                if(copyTextToClipboard(profileUrl)){
+                  toast("URL copied to clipboard", { type: "success" });
```

Comment:
Should toast messages also be translated?

Reply:
Did the change